fairmotion/tasks/clustering/features/manual.py

```diff
@@ -36,7 +37,16 @@ def __init__(self, motion, skeleton=feat_utils.Skeleton.PFNN):
         )
 
     def next_frame(self):
-        self.frame_num += 1
+        if self.frame_num + 1 > self.num_frames:
+            print("Reached end of motion!")
```

Comment:
Between printing error and returning explicit 'end_of_frame' value, what would be better?